streamspy/streamspy.py

```diff
@@ -0,0 +1,24 @@
+from gymnasium import Env
+from typing import Tuple, Any, Optional
+
+ActType = Any
+ObsType = Any
+RenderFrame = Any
+
+class StreamsSolver(Env):
```

Comment:
framework for creating a gym environment with `gymnasium`. these are all placeholders